tests/p2p_connection_test.rs

```diff
@@ -15,119 +15,45 @@ mod p2p_connection_test {
     use uniffi_lipalightninglib::config::NodeAddress;
 
     const NIGIRI_LND_ADDR: &str = "127.0.0.1:9735";
-    const FAULTY_ADDR: &str = "127.0.0.1:9";
 
     #[test]
     fn test_p2p_connection() {
         setup::nigiri::start();
 
-        // test successful p2p connection
-        {
-            let lsp_info = setup::nigiri::query_node_info(NodeInstance::NigiriLnd).unwrap();
-            let lsp_node = NodeAddress {
-                pub_key: lsp_info.pub_key,
-                host: NIGIRI_LND_ADDR.to_string(),
-            };
+        let lsp_info = setup::nigiri::query_node_info(NodeInstance::NigiriLnd).unwrap();
+        let lsp_node = NodeAddress {
+            pub_key: lsp_info.pub_key,
+            host: NIGIRI_LND_ADDR.to_string(),
+        };
 
-            let node = NodeHandle::new(lsp_node.clone()).start().unwrap();
+        let node = NodeHandle::new(lsp_node.clone()).start().unwrap();
 
+        // Test successful p2p connection.
+        {
+            sleep(Duration::from_secs(100));
```

Comment:
We should try avoid these long periods of sleep whenever possible. Would you happen to know why it's needed here? If it's because of the `TaskPeriods` we could have a test configuration that makes tasks run more frequently.